mailboxes/lang/de

```diff
@@ -233,73 +234,73 @@ send_efrom=Fehlende Absenderadresse (FROM)
 send_esubject=Fehlender Betreff
 send_title=Mail versandt
 send_title2=Mail gesichert
-send_ok=Mail erfolgreich versandt an $1
-send_sending=Sende Mail an $1 ..
-send_ecannot=Sie dürfen im Namen dieses Benutzers keine Mail versenden
+send_ok=Mail erfolgreich an $1 versendet
+send_sending=Sende Mail an $1
+send_ecannot=Sie dürfen Mails im Namen dieses Benutzers nicht versenden
 send_esmtp=SMTP-Kommando $1 gescheitert : $2
-send_eattach=Die Dateianhänge dürfen zusammen nicht mehr als $1 KiB groß sein.
+send_eattach=Die Dateianhänge dürfen zusammen nicht mehr als $1 KB groß sein
 send_eperms=Benutzer $1 kann $2 nicht lesen
-send_eperms2=Sie dürfen die Datei $1 nicht versenden
-send_epath=Das Sendmail-Binary $1 existiert nicht.
+send_eperms2=Die Datei $1 darf nicht versendet werden
+send_epath=Das Sendmail-Binary $1 existiert nicht
 send_efile=Konnte Dateianhänge nicht lesen $1 : $2
-send_done=.. fertig.
+send_done=Fertig.
 
-send_epass=Sie können diese Mail nicht signieren, da Sie noch keine Passphrase im GnuPG-Modul hinterlegt haben.
+send_epass=Diese Mail kann nicht signiert werden, da keine Passphrase im GnuPG-Modul hinterlegt wurde
 send_esign=Konnte Mail nicht signieren : $1
 send_ekey=Konnte den Schlüssel für die Mail-Adresse $1 nicht finden
 send_ecrypt=Konnte Mail nicht verschlüsseln : $1
 send_eword=Falsch geschriebenes Wort $1
 send_eword2=Falsch geschriebenes Wort $1 - Mögliche Korrektur $2
 send_eline=In Zeile $1:
-send_espell=Die folgenden Rechtschreibefehler wurden in Ihrer Mail gefunden ..
-send_draft=Die Mail an $1 wurde als Entwurf gespeichert.
-send_drafting=Speichere Mail für $1 in Entwurfsverzeichnis
-send_eattachsize=Die Dateianhänge überschreiten die maximal erlaubte Größe von $1 bytes.
+send_espell=Die folgenden Rechtschreibfehler wurden gefunden
+send_draft=Die Mail an $1 wurde als Entwurf gespeichert
+send_drafting=Mail für $1 in Entwurfsverzeichnis speichern
+send_eattachsize=Die Dateianhänge überschreiten die maximal erlaubte Größe von $1 Bytes
 
 delete_title=Mail löschen
-delete_rusure=Sind Sie sicher, daß Sie die ausgewählten $1 Mails aus/von $2 löschen wollen? Aufgrund der Größe und des Formates dieser Mailbox wird dies ein wenig dauern. Während des Löschvorganges sollten keine anderen Aktionen stattfinden. Bitte ein wenig Geduld ...
-delete_rusure2=Sind Sie sicher, daß Sie diese Mail aus/von $1 löschen wollen? Aufgrund der Größe und des Formates dieser Mailbox wird dies ein wenig dauern. Während des Löschvorganges sollten keine anderen Aktionen stattfinden. Bitte ein wenig Geduld ...
+delete_rusure=Sollen die ausgewählten $1 Mails aus/von $2 gelöscht werden? Aufgrund der Größe und des Formats dieser Mailbox kann dies einige Zeit in Anspruch nehmen…
+delete_rusure2=Soll die Mail aus/von $1 gelöscht werden? Aufgrund der Größe und des Formates dieser Mailbox kann dies einige Zeit in Anspruch nehmen. Während des Löschvorganges sollten keine anderen Aktionen stattfinden. Bitte Geduld…
```

Comment:
You should never use ellipsis, i.e. `…`.

And, all these phrases should end just with a regular stop, i.e. `.`, as the original `lang/en` file.